internal/pkg/cli/init.go

```diff
@@ -553,6 +553,7 @@ func (o *initOpts) askEnvNameAndMaybeInit() error {
 		}
 		// Customer has selected an existing environment. Return early.
 		if selectedEnv != envPromptCreateNew {
+			o.envName = &selectedEnv
```

Comment:
It seems like o.envName is meant to be a pointer to `initEnvCmd.name`. Instead of changing the value of `o.envName`, what do you think of changing the value that it points to (i.e. initEnvCmd.name):
```go
# Either
initEnvCmd.Name = selectedEnv

# Or
*(o.envName) = selectedEnv
```

Reply:
I agree with this yeah!. Looking through it I was a bit confused what the point of the pointer is. Of these two options I'd rather change where `o.envName` is pointing because we'd have to pull the variable out of the interface of `initEnvCmd` to change the name here, and I'd rather avoid this since it's not guaranteed to be of type `initEnvOpts` here.

Reply:
Apparently this causes a potential nil pointer de-reference. I'll have to look further into this for now.